app/views/layouts/application.html.erb

```diff
@@ -40,7 +41,7 @@
           <% end %>
         </div>
         <div class="govuk-header__content">
-          <%= link_to "Rails DfE Boilerplate", "/", class: "govuk-header__link govuk-header__link--service-name" %>
+          <%= link_to "GOVU.UK Rails Boilerplate", "/", class: "govuk-header__link govuk-header__link--service-name" %>
```

Comment:
Typo here

Reply:
Thanks @peteryates. Fixed and force pushed 👌